syntax="proto3";
package MQTT;


service MQTTService {
  rpc publish(PublishRequest) returns (NoParam);
};


message NoParam {};

message PublishRequest {
  string topic=1;
  string msg=2;
};